### PR TITLE
Update GroupBys In Cloud Aggregate Table

### PR DESCRIPTION
**Pull request by chakrabortyr**
## Description
Added missing group by ids to the aggregate table generation.

## Motivation and Context
To ensure our drilldowns/group bys in the portal work properly

## Tests performed
Ran regression + integration tests in Docker to make sure those didn't break.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json [hunk 1]
```diff
@@ -50,8 +50,11 @@
               "${AGGREGATION_UNIT}_id",
               "year",
               "${AGGREGATION_UNIT}",
+              "host_resource_id",
               "configuration",
```
Comment (jpwhite4) [inline configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json]: Which group_by uses the 'configuration' column?

File: configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json [hunk 2]
```diff
@@ -50,8 +50,11 @@
...
               "processorbucket_id",
+              "memorybucket_id",
+              "instance_type_id",
```
Comment (jpwhite4) [inline configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json]: Missing submission_venue grouping

File: configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json (hunk 1 above)
Comment (chakrabortyr) [reply configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json]: I suspect "Group By Instance Type" may have at one point and this was not removed when it changed. I've now done so.

Comment (chakrabortyr) [reply configuration/etl/etl_action_defs.d/cloud_common/cloud_metrics_aggregation.json]: Come to think of it, I don't think we should have "configuration" (or anything varchar) in the aggregate tables at all. I'll have a look at removing this in another PR.

**Pull request merged**